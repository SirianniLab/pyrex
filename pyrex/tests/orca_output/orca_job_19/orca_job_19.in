!rhf sto-3g 


*xyz 0 1
C    0.134698715505   -0.151025363114    0.037941882147
O   -1.155678141712   -0.252214795328   -0.029699630186
O    1.081320831201    0.555607655622   -0.033146926538
H    0.363380814931   -1.613747456079    0.331719215176
H   -0.787110866776   -1.403084278070    0.213933755993
*


