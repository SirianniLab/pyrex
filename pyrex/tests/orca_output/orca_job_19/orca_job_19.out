
                                 *****************
                                 * O   R   C   A *
                                 *****************

           --- An Ab Initio, DFT and Semiempirical electronic structure package ---

                  #######################################################
                  #                        -***-                        #
                  #          Department of theory and spectroscopy      #
                  #               Directorship: Frank Neese             #
                  #        Max Planck Institute fuer Kohlenforschung    #
                  #                Kaiser Wilhelm Platz 1               #
                  #                 D-45470 Muelheim/Ruhr               #
                  #                      Germany                        #
                  #                                                     #
                  #                  All rights reserved                #
                  #                        -***-                        #
                  #######################################################


                         Program Version 4.2.0 -  RELEASE  -


 With contributions from (in alphabetic order):
   Daniel Aravena         : Magnetic Suceptibility
   Michael Atanasov       : Ab Initio Ligand Field Theory (pilot matlab implementation)
   Alexander A. Auer      : GIAO ZORA, VPT2
   Ute Becker             : Parallelization
   Giovanni Bistoni       : ED, misc. LED, open-shell LED, HFLED
   Martin Brehm           : Molecular dynamics
   Dmytro Bykov           : SCF Hessian
   Vijay G. Chilkuri      : MRCI spin determinant printing, contributions to CSF-ICE
   Dipayan Datta          : RHF DLPNO-CCSD density
   Achintya Kumar Dutta   : EOM-CC, STEOM-CC
   Dmitry Ganyushin       : Spin-Orbit,Spin-Spin,Magnetic field MRCI
   Miquel Garcia          : C-PCM Hessian, Gaussian charge scheme
   Yang Guo               : DLPNO-NEVPT2, CIM, IAO-localization
   Andreas Hansen         : Spin unrestricted coupled pair/coupled cluster methods
   Benjamin Helmich-Paris : CASSCF linear response (MC-RPA)
   Lee Huntington         : MR-EOM, pCC
   Robert Izsak           : Overlap fitted RIJCOSX, COSX-SCS-MP3, EOM
   Christian Kollmar      : KDIIS, OOCD, Brueckner-CCSD(T), CCSD density
   Simone Kossmann        : Meta GGA functionals, TD-DFT gradient, OOMP2, MP2 Hessian
   Martin Krupicka        : AUTO-CI
   Lucas Lang             : DCDCAS
   Dagmar Lenk            : GEPOL surface, SMD
   Dimitrios Liakos       : Extrapolation schemes; Compound Job, initial MDCI parallelization
   Dimitrios Manganas     : Further ROCIS development; embedding schemes
   Dimitrios Pantazis     : SARC Basis sets
   Taras Petrenko         : DFT Hessian,TD-DFT gradient, ASA, ECA, R-Raman, ABS, FL, XAS/XES, NRVS
   Peter Pinski           : DLPNO-MP2, DLPNO-MP2 Gradient
   Christoph Reimann      : Effective Core Potentials
   Marius Retegan         : Local ZFS, SOC
   Christoph Riplinger    : Optimizer, TS searches, QM/MM, DLPNO-CCSD(T), (RO)-DLPNO pert. Triples
   Tobias Risthaus        : Range-separated hybrids, TD-DFT gradient, RPA, STAB
   Michael Roemelt        : Original ROCIS implementation
   Masaaki Saitow         : Open-shell DLPNO-CCSD energy and density
   Barbara Sandhoefer     : DKH picture change effects
   Avijit Sen             : IP-ROCIS
   Kantharuban Sivalingam : CASSCF convergence, NEVPT2, FIC-MRCI
   Bernardo de Souza      : ESD, SOC TD-DFT
   Georgi Stoychev        : AutoAux, RI-MP2 NMR
   Willem Van den Heuvel  : Paramagnetic NMR
   Boris Wezisla          : Elementary symmetry handling
   Frank Wennmohs         : Technical directorship


 We gratefully acknowledge several colleagues who have allowed us to
 interface, adapt or use parts of their codes:
   Stefan Grimme, W. Hujo, H. Kruse,             : VdW corrections, initial TS optimization,
                  C. Bannwarth                     DFT functionals, gCP, sTDA/sTD-DF
   Ed Valeev, F. Pavosevic, A. Kumar             : LibInt (2-el integral package), F12 methods
   Garnet Chan, S. Sharma, J. Yang, R. Olivares  : DMRG
   Ulf Ekstrom                                   : XCFun DFT Library
   Mihaly Kallay                                 : mrcc  (arbitrary order and MRCC methods)
   Andreas Klamt, Michael Diedenhofen            : otool_cosmo (COSMO solvation model)
   Jiri Pittner, Ondrej Demel                    : Mk-CCSD
   Frank Weinhold                                : gennbo (NPA and NBO analysis)
   Christopher J. Cramer and Donald G. Truhlar   : smd solvation model
   Lars Goerigk                                  : TD-DFT with DH, B97 family of functionals
   V. Asgeirsson, H. Jonsson                     : NEB implementation
   FAccTs GmbH                                   : IRC, NEB, NEB-TS, Multilevel, MM, QM/MM, CI optimization
   S Lehtola, MJT Oliveira, MAL Marques          : LibXC Library


 Your calculation uses the libint2 library for the computation of 2-el integrals
 For citations please refer to: http://libint.valeyev.net

 Your ORCA version has been built with support for libXC version: 4.2.3
 For citations please refer to: https://tddft.org/programs/libxc/

 This ORCA versions uses:
   CBLAS   interface :  Fast vector & matrix operations
   LAPACKE interface :  Fast linear algebra routines
   SCALAPACK package :  Parallel linear algebra routines


----- Orbital basis set information -----
Your calculation utilizes the basis: STO-3G
   H-Ne       : W. J. Hehre, R. F. Stewart and J. A. Pople, J. Chem. Phys. 2657 (1969).
   Na-Ar      : W. J. Hehre, R. Ditchfield, R. F. Stewart and J. A. Pople, J. Chem. Phys. 2769 (1970).
   K,Ca,Ga-Kr : W. J. Pietro, B. A. Levy, W. J. Hehre and R. F. Stewart, J. Am. Chem. Soc. 19, 2225 (1980).
   Sc-Zn,Y-Cd : W. J. Pietro and W. J. Hehre, J. Comp. Chem. 4, 241 (1983).

================================================================================
                                        WARNINGS
                       Please study these warnings very carefully!
================================================================================


INFO   : the flag for use of LIBINT has been found!

================================================================================
                                       INPUT FILE
================================================================================
NAME = /home/wderricotte/src/pyrex/pyrex/tests/orca_output/orca_job_19/orca_job_19.in
|  1> !rhf sto-3g 
|  2> 
|  3> 
|  4> *xyz 0 1
|  5> C    0.134698715505   -0.151025363114    0.037941882147
|  6> O   -1.155678141712   -0.252214795328   -0.029699630186
|  7> O    1.081320831201    0.555607655622   -0.033146926538
|  8> H    0.363380814931   -1.613747456079    0.331719215176
|  9> H   -0.787110866776   -1.403084278070    0.213933755993
| 10> *
| 11> 
| 12> 
| 13> 
| 14>                          ****END OF INPUT****
================================================================================

                       ****************************
                       * Single Point Calculation *
                       ****************************

---------------------------------
CARTESIAN COORDINATES (ANGSTROEM)
---------------------------------
  C      0.134699   -0.151025    0.037942
  O     -1.155678   -0.252215   -0.029700
  O      1.081321    0.555608   -0.033147
  H      0.363381   -1.613747    0.331719
  H     -0.787111   -1.403084    0.213934

----------------------------
CARTESIAN COORDINATES (A.U.)
----------------------------
  NO LB      ZA    FRAG     MASS         X           Y           Z
   0 C     6.0000    0    12.011    0.254544   -0.285397    0.071700
   1 O     8.0000    0    15.999   -2.183915   -0.476617   -0.056124
   2 O     8.0000    0    15.999    2.043400    1.049946   -0.062639
   3 H     1.0000    0     1.008    0.686690   -3.049541    0.626858
   4 H     1.0000    0     1.008   -1.487424   -2.651445    0.404276

--------------------------------
INTERNAL COORDINATES (ANGSTROEM)
--------------------------------
 C      0   0   0     0.000000000000     0.00000000     0.00000000
 O      1   0   0     1.296104590310     0.00000000     0.00000000
 O      1   2   0     1.183417623586   147.12461735     0.00000000
 H      1   2   3     1.509356334749    94.89546963   180.03111473
 H      4   1   2     1.175535409352    70.03455742     0.00000000

---------------------------
INTERNAL COORDINATES (A.U.)
---------------------------
 C      0   0   0     0.000000000000     0.00000000     0.00000000
 O      1   0   0     2.449282716605     0.00000000     0.00000000
 O      1   2   0     2.236335210633   147.12461735     0.00000000
 H      1   2   3     2.852270111175    94.89546963   180.03111473
 H      4   1   2     2.221439984403    70.03455742     0.00000000

---------------------
BASIS SET INFORMATION
---------------------
There are 3 groups of distinct atoms

 Group   1 Type C   : 6s3p contracted to 2s1p pattern {33/3}
 Group   2 Type O   : 6s3p contracted to 2s1p pattern {33/3}
 Group   3 Type H   : 3s contracted to 1s pattern {3}

Atom   0C    basis set group =>   1
Atom   1O    basis set group =>   2
Atom   2O    basis set group =>   2
Atom   3H    basis set group =>   3
Atom   4H    basis set group =>   3

Checking for AutoStart:
The File: /home/wderricotte/src/pyrex/pyrex/tests/orca_output/orca_job_19/orca_job_19.gbw exists
Trying to determine its content:
     ... Fine, the file contains calculation information
     ... Fine, the calculation information was read
     ... Fine, the file contains a basis set
     ... Fine, the basis set was read
     ... Fine, the file contains a geometry
     ... Fine, the geometry was read
     ... Fine, the file contains a set of orbitals
     ... Fine, the orbitals can be read
     => possible old guess file was deleted
     => GBW file was renamed to GES file
     => GES file is set as startup file
     => Guess is set to MORead
     ... now leaving AutoStart

------------------------------------------------------------------------------
                           ORCA GTO INTEGRAL CALCULATION
------------------------------------------------------------------------------

                         BASIS SET STATISTICS AND STARTUP INFO

 # of primitive gaussian shells          ...   33
 # of primitive gaussian functions       ...   51
 # of contracted shells                  ...   11
 # of contracted basis functions         ...   17
 Highest angular momentum                ...    1
 Maximum contraction depth               ...    3
 Integral package used                   ... LIBINT
 Integral threshhold            Thresh   ...  1.000e-10
 Primitive cut-off              TCut     ...  1.000e-11


------------------------------ INTEGRAL EVALUATION ----------------------------


 * One electron integrals 
 Pre-screening matrix                    ... done
 Shell pair data                         ... done (   0.000 sec)

-------------------------------------------------------------------------------
                                 ORCA SCF
-------------------------------------------------------------------------------

------------
SCF SETTINGS
------------
Hamiltonian:
 Ab initio Hamiltonian  Method          .... Hartree-Fock(GTOs)


General Settings:
 Integral files         IntName         .... /home/wderricotte/src/pyrex/pyrex/tests/orca_output/orca_job_19/orca_job_19
 Hartree-Fock type      HFTyp           .... RHF
 Total Charge           Charge          ....    0
 Multiplicity           Mult            ....    1
 Number of Electrons    NEL             ....   24
 Basis Dimension        Dim             ....   17
 Nuclear Repulsion      ENuc            ....     68.7482063975 Eh

Convergence Acceleration:
 DIIS                   CNVDIIS         .... on
   Start iteration      DIISMaxIt       ....    12
   Startup error        DIISStart       ....  0.200000
   # of expansion vecs  DIISMaxEq       ....     5
   Bias factor          DIISBfac        ....   1.050
   Max. coefficient     DIISMaxC        ....  10.000
 Newton-Raphson         CNVNR           .... off
 SOSCF                  CNVSOSCF        .... on
   Start iteration      SOSCFMaxIt      ....   150
   Startup grad/error   SOSCFStart      ....  0.003300
 Level Shifting         CNVShift        .... on
   Level shift para.    LevelShift      ....    0.2500
   Turn off err/grad.   ShiftErr        ....    0.0010
 Zerner damping         CNVZerner       .... off
 Static damping         CNVDamp         .... on
   Fraction old density DampFac         ....    0.7000
   Max. Damping (<1)    DampMax         ....    0.9800
   Min. Damping (>=0)   DampMin         ....    0.0000
   Turn off err/grad.   DampErr         ....    0.1000
 Fernandez-Rico         CNVRico         .... off

SCF Procedure:
 Maximum # iterations   MaxIter         ....   125
 SCF integral mode      SCFMode         .... Direct
   Integral package                     .... LIBINT
 Reset frequency        DirectResetFreq ....    20
 Integral Threshold     Thresh          ....  1.000e-10 Eh
 Primitive CutOff       TCut            ....  1.000e-11 Eh

Convergence Tolerance:
 Convergence Check Mode ConvCheckMode   .... Total+1el-Energy
 Convergence forced     ConvForced      .... 0
 Energy Change          TolE            ....  1.000e-06 Eh
 1-El. energy change                    ....  1.000e-03 Eh
 Orbital Gradient       TolG            ....  5.000e-05
 Orbital Rotation angle TolX            ....  5.000e-05
 DIIS Error             TolErr          ....  1.000e-06


Diagonalization of the overlap matrix:
Smallest eigenvalue                        ... 2.276e-01
Time for diagonalization                   ...    0.005 sec
Threshold for overlap eigenvalues          ... 1.000e-08
Number of eigenvalues below threshold      ... 0
Time for construction of square roots      ...    0.004 sec
Total time needed                          ...    0.010 sec

---------------------
INITIAL GUESS: MOREAD
---------------------
Guess MOs are being read from file: /home/wderricotte/src/pyrex/pyrex/tests/orca_output/orca_job_19/orca_job_19.ges
Input Geometry matches current geometry (good)
Input basis set matches current basis set (good)
MOs were renormalized
MOs were reorthogonalized (Cholesky)
                      ------------------
                      INITIAL GUESS DONE (   0.0 sec)
                      ------------------
--------------
SCF ITERATIONS
--------------
ITER       Energy         Delta-E        Max-DP      RMS-DP      [F,P]     Damp
               ***  Starting incremental Fock matrix formation  ***
                      *** Initiating the SOSCF procedure ***
                      *** Re-Reading the Fockian *** 
                      *** Removing any level shift *** 
ITER      Energy       Delta-E        Grad      Rot      Max-DP    RMS-DP
  0   -186.01232335 -186.0123233487  0.000001  0.000001  0.000002  0.000000
               *** Restarting incremental Fock matrix formation ***
                 **** Energy Check signals convergence ****
              ***Rediagonalizing the Fockian in SOSCF/NRSCF***

               *****************************************************
               *                     SUCCESS                       *
               *           SCF CONVERGED AFTER   1 CYCLES          *
               *****************************************************


----------------
TOTAL SCF ENERGY
----------------

Total Energy       :         -186.01232335 Eh           -5061.65265 eV

Components:
Nuclear Repulsion  :           68.74820640 Eh            1870.73380 eV
Electronic Energy  :         -254.76052975 Eh           -6932.38645 eV
One Electron Energy:         -392.81934911 Eh          -10689.15792 eV
Two Electron Energy:          138.05881936 Eh            3756.77147 eV

Virial components:
Potential Energy   :         -370.67422249 Eh          -10086.55838 eV
Kinetic Energy     :          184.66189914 Eh            5024.90574 eV
Virial Ratio       :            2.00731296


---------------
SCF CONVERGENCE
---------------

  Last Energy change         ...   -7.1338e-12  Tolerance :   1.0000e-06
  Last MAX-Density change    ...    1.7616e-06  Tolerance :   1.0000e-05
  Last RMS-Density change    ...    2.7692e-07  Tolerance :   1.0000e-06
  Last Orbital Gradient      ...    8.1874e-07  Tolerance :   5.0000e-05
  Last Orbital Rotation      ...    1.4581e-06  Tolerance :   5.0000e-05

             **** THE GBW FILE WAS UPDATED (/home/wderricotte/src/pyrex/pyrex/tests/orca_output/orca_job_19/orca_job_19.gbw) ****
             **** DENSITY FILE WAS UPDATED (/home/wderricotte/src/pyrex/pyrex/tests/orca_output/orca_job_19/orca_job_19.scfp) ****
             **** ENERGY FILE WAS UPDATED (/home/wderricotte/src/pyrex/pyrex/tests/orca_output/orca_job_19/orca_job_19.en.tmp) ****
             **** THE GBW FILE WAS UPDATED (/home/wderricotte/src/pyrex/pyrex/tests/orca_output/orca_job_19/orca_job_19.gbw) ****
             **** DENSITY FILE WAS UPDATED (/home/wderricotte/src/pyrex/pyrex/tests/orca_output/orca_job_19/orca_job_19.scfp) ****
----------------
ORBITAL ENERGIES
----------------

  NO   OCC          E(Eh)            E(eV) 
   0   2.0000     -20.345335      -553.6247 
   1   2.0000     -20.301361      -552.4281 
   2   2.0000     -11.252702      -306.2016 
   3   2.0000      -1.417811       -38.5806 
   4   2.0000      -1.338947       -36.4346 
   5   2.0000      -0.721498       -19.6330 
   6   2.0000      -0.674128       -18.3439 
   7   2.0000      -0.586102       -15.9487 
   8   2.0000      -0.578282       -15.7359 
   9   2.0000      -0.458761       -12.4835 
  10   2.0000      -0.375243       -10.2109 
  11   2.0000      -0.300687        -8.1821 
  12   0.0000       0.260809         7.0970 
  13   0.0000       0.286650         7.8001 
  14   0.0000       0.525467        14.2987 
  15   0.0000       0.628799        17.1105 
  16   0.0000       1.082357        29.4524 

                    ********************************
                    * MULLIKEN POPULATION ANALYSIS *
                    ********************************

-----------------------
MULLIKEN ATOMIC CHARGES
-----------------------
   0 C :    0.412645
   1 O :   -0.290950
   2 O :   -0.218448
   3 H :   -0.191484
   4 H :    0.288237
Sum of atomic charges:    0.0000000

--------------------------------
MULLIKEN REDUCED ORBITAL CHARGES
--------------------------------
  0 C s       :     3.097013  s :     3.097013
      pz      :     0.861335  p :     2.490342
      px      :     0.838237
      py      :     0.790770
  1 O s       :     3.870101  s :     3.870101
      pz      :     1.731918  p :     4.420848
      px      :     1.198140
      py      :     1.490791
  2 O s       :     3.858796  s :     3.858796
      pz      :     1.406276  p :     4.359653
      px      :     1.437013
      py      :     1.516364
  3 H s       :     1.191484  s :     1.191484
  4 H s       :     0.711763  s :     0.711763


                     *******************************
                     * LOEWDIN POPULATION ANALYSIS *
                     *******************************

----------------------
LOEWDIN ATOMIC CHARGES
----------------------
   0 C :    0.335224
   1 O :   -0.249413
   2 O :   -0.149032
   3 H :   -0.195582
   4 H :    0.258804

-------------------------------
LOEWDIN REDUCED ORBITAL CHARGES
-------------------------------
  0 C s       :     3.068365  s :     3.068365
      pz      :     0.867330  p :     2.596411
      px      :     0.910942
      py      :     0.818139
  1 O s       :     3.764073  s :     3.764073
      pz      :     1.729313  p :     4.485340
      px      :     1.253126
      py      :     1.502902
  2 O s       :     3.732289  s :     3.732289
      pz      :     1.404298  p :     4.416742
      px      :     1.471188
      py      :     1.541257
  3 H s       :     1.195582  s :     1.195582
  4 H s       :     0.741196  s :     0.741196


                      *****************************
                      * MAYER POPULATION ANALYSIS *
                      *****************************

  NA   - Mulliken gross atomic population
  ZA   - Total nuclear charge
  QA   - Mulliken gross atomic charge
  VA   - Mayer's total valence
  BVA  - Mayer's bonded valence
  FA   - Mayer's free valence

  ATOM       NA         ZA         QA         VA         BVA        FA
  0 C      5.5874     6.0000     0.4126     3.8889     3.8889     0.0000
  1 O      8.2909     8.0000    -0.2909     2.2088     2.2088     0.0000
  2 O      8.2184     8.0000    -0.2184     2.3920     2.3920     0.0000
  3 H      1.1915     1.0000    -0.1915     0.9633     0.9633     0.0000
  4 H      0.7118     1.0000     0.2882     0.9169     0.9169     0.0000

  Mayer bond orders larger than 0.100000
B(  0-C ,  1-O ) :   1.2496 B(  0-C ,  2-O ) :   2.0184 B(  0-C ,  3-H ) :   0.5224 
B(  1-O ,  2-O ) :   0.2077 B(  1-O ,  3-H ) :   0.1660 B(  1-O ,  4-H ) :   0.5855 
B(  2-O ,  3-H ) :   0.1040 B(  3-H ,  4-H ) :   0.1710 

-------
TIMINGS
-------

Total SCF time: 0 days 0 hours 0 min 0 sec 

Total time                  ....       0.932 sec
Sum of individual times     ....       0.255 sec  ( 27.4%)

Fock matrix formation       ....       0.205 sec  ( 22.0%)
Diagonalization             ....       0.000 sec  (  0.0%)
Density matrix formation    ....       0.000 sec  (  0.0%)
Population analysis         ....       0.035 sec  (  3.7%)
Initial guess               ....       0.003 sec  (  0.3%)
Orbital Transformation      ....       0.000 sec  (  0.0%)
Orbital Orthonormalization  ....       0.000 sec  (  0.0%)
DIIS solution               ....       0.000 sec  (  0.0%)
SOSCF solution              ....       0.012 sec  (  1.3%)

-------------------------   --------------------
FINAL SINGLE POINT ENERGY      -186.012323348689
-------------------------   --------------------


                            ***************************************
                            *     ORCA property calculations      *
                            ***************************************

                                    ---------------------
                                    Active property flags
                                    ---------------------
   (+) Dipole Moment


------------------------------------------------------------------------------
                       ORCA ELECTRIC PROPERTIES CALCULATION
------------------------------------------------------------------------------

Dipole Moment Calculation                       ... on
Quadrupole Moment Calculation                   ... off
Polarizability Calculation                      ... off
GBWName                                         ... /home/wderricotte/src/pyrex/pyrex/tests/orca_output/orca_job_19/orca_job_19.gbw
Electron density file                           ... /home/wderricotte/src/pyrex/pyrex/tests/orca_output/orca_job_19/orca_job_19.scfp
The origin for moment calculation is the CENTER OF MASS  = ( 0.000045, -0.000039  0.000011)

-------------
DIPOLE MOMENT
-------------
                                X             Y             Z
Electronic contribution:      0.13804       2.31838      -0.43195
Nuclear contribution   :     -0.39868      -2.82579       0.51098
                        -----------------------------------------
Total Dipole Moment    :     -0.26064      -0.50741       0.07903
                        -----------------------------------------
Magnitude (a.u.)       :      0.57589
Magnitude (Debye)      :      1.46379



--------------------
Rotational spectrum 
--------------------
 
Rotational constants in cm-1:     3.259252     0.359252     0.323584 
Rotational constants in MHz : 97709.929957 10770.091131  9700.817283 

 Dipole components along the rotational axes: 
x,y,z [a.u.] :    -0.432289    -0.380489    -0.000075 
x,y,z [Debye]:    -1.098791    -0.967127    -0.000190 

 

Timings for individual modules:

Sum of individual times         ...        2.037 sec (=   0.034 min)
GTO integral calculation        ...        0.904 sec (=   0.015 min)  44.4 %
SCF iterations                  ...        1.133 sec (=   0.019 min)  55.6 %
                             ****ORCA TERMINATED NORMALLY****
TOTAL RUN TIME: 0 days 0 hours 0 minutes 2 seconds 722 msec
